askama/Cargo.toml

```diff
@@ -24,13 +24,13 @@ urlencode = ["askama_shared/percent-encoding"]
 serde-json = ["askama_shared/json"]
 serde-yaml = ["askama_shared/yaml"]
 num-traits = ["askama_shared/num-traits"]
-with-actix-web = ["askama_shared/actix-web"]
-with-axum = ["askama_shared/axum"]
-with-gotham = ["askama_shared/gotham"]
-with-mendes = ["askama_shared/mendes"]
-with-rocket = ["askama_shared/rocket"]
-with-tide = ["askama_shared/tide"]
-with-warp = ["askama_shared/warp"]
+with-actix-web = ["askama_derive/with-actix-web"]
+with-axum = ["askama_derive/with-axum"]
+with-gotham = ["askama_derive/with-gotham"]
+with-mendes = ["askama_derive/with-mendes"]
+with-rocket = ["askama_derive/with-rocket"]
+with-tide = ["askama_derive/with-tide"]
+with-warp = ["askama_derive/with-warp"]
```

Comment:
It makes sense to me that these feature dependencies are needed, same goes for the ones in askama_derive. However, all of the other changes of this PR seem like a distraction at best, and I don't really get why you reverted all of my changes from #657.